future-apps/staking/contracts/ERCStaking.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity ^0.4.18;
+
+
+interface ERCStaking {
+  event Staked(address indexed user, uint256 amount, uint256 total, bytes data);
+  event Unstaked(address indexed user, uint256 amount, uint256 total, bytes data);
+
+  function stake(uint256 amount, bytes data) public;
+  function stakeFor(address user, uint256 amount, bytes data) public;
+  function unstake(uint256 amount, bytes data) public;
+
+  function totalStakedFor(address addr) public view returns (uint256);
```

Comment:
Missing `totalStaked`? I saw discussion [here](https://github.com/ethereum/EIPs/issues/900#issuecomment-368108039) and [your response](https://github.com/ethereum/EIPs/issues/900#issuecomment-368109493), but it seems it's part of the interface definition.